liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -734,9 +736,13 @@ private void addDomPropertyIfNotFound(Xpp3Dom sysProps, String key, String value
                     elements.add(element(name("looseApplication"), "true"));
                     elements.add(element(name("stripVersion"), "true"));
                     elements.add(element(name("installAppPackages"), "project"));
-                    elements.add(element(name("configFile"), configFile.getCanonicalPath()));
-                } else if (goal.equals("create-server")) {
-                    elements.add(element(name("configFile"), configFile.getCanonicalPath()));
+                    if (serverXmlFile != null) {
+                        elements.add(element(name("configFile"), serverXmlFile.getCanonicalPath()));
```

Comment:
Why are these elements named `configFile` and not `serverXmlFile`?

Reply:
Isn't that for the liberty-plugin-config.xml file? I didn't change what gets written there.